### PR TITLE
Implement docker build/push in CICD

### LINKED ISSUE #6: Create the docker image and push it to docker hub as part of the CICD
## 💡 Description

As what is done for the other registry components (registry-loader, registry-api for example).

### PR DESCRIPTION
**Pull request by alexdunnjpl**
## ⚙️ Test Data and/or Report
n/a

## ♻️ Related Issues
fixes #6

### CODE REVIEW

File: .github/workflows/stable-cicd.yaml [hunk 1]
```diff
@@ -73,6 +73,27 @@ jobs:
...
+                    username: ${{secrets.DOCKERHUB_USERNAME}}
+                    password: ${{secrets.DOCKERHUB_TOKEN}}
+            -
+                name: 🎰 QEMU Multiple Machine Emulation
+                uses: docker/setup-qemu-action@v2
+            -
+                name: 🚢 Docker Buildx
+                uses: docker/setup-buildx-action@v2
+            -
+              name: 🧱 Image Construction and Publication
```
Comment (nutjob4life) [inline .github/workflows/stable-cicd.yaml:89]: Indentation looks a little off here (previously 4 spaces, suddenly 2). Not a deal-breaker, though.

File: .github/workflows/stable-cicd.yaml [hunk 2]
```diff
@@ -73,6 +73,27 @@ jobs:
...
+              uses: docker/build-push-action@v3
+              with:
+                  context: ./
+                  file: ./docker/Dockerfile
+                  platforms: linux/amd64,linux/arm64
+                  push: true
+                  tags: ${{secrets.DOCKERHUB_USERNAME}}/registry-api-sweepers:latest
```
Comment (nutjob4life) [inline .github/workflows/stable-cicd.yaml:96]: Can we get a version number here instead of `:latest`?

File: .github/workflows/unstable-cicd.yaml [hunk 3]
```diff
@@ -78,6 +78,27 @@ jobs:
...
+                    username: ${{secrets.DOCKERHUB_USERNAME}}
+                    password: ${{secrets.DOCKERHUB_TOKEN}}
+            -
+                name: 🎰 QEMU Multiple Machine Emulation
+                uses: docker/setup-qemu-action@v2
+            -
+                name: 🚢 Docker Buildx
+                uses: docker/setup-buildx-action@v2
+            -
+              name: 🧱 Image Construction and Publication
```
Comment (nutjob4life) [inline .github/workflows/unstable-cicd.yaml:94]: Same indentation issue

**Review by nutjob4life — CHANGES_REQUESTED** [review]

**Comment by alexdunnjpl** [comment]
@nutjob4life so you don't think I'm totally crazy, this'll all get ironed out as I finish it - I opened the PR before I realized I should just do it all on `main` and revert later.

**Pull request merged**